lib/rubocop/version.rb

```diff
@@ -39,6 +39,21 @@ def self.version(debug: false, env: nil)
       end
     end
 
+    # @api private
+    def self.parser_version
+      config_path = ConfigFinder.find_config_path(Dir.pwd)
+      yaml = YAML.safe_load(
+        File.read(config_path), permitted_classes: [Regexp, Symbol], aliases: true
+      )
+
+      if yaml.dig('AllCops', 'ParserEngine') == 'parser_prism'
+        require 'prism'
+        "Prism #{Prism::VERSION}"
```

Comment:
The parser translation is now bundled with Prism, so I don't think mentioning it separately makes much sense. The multiple parser engines are something temporary - this will disappear once we switch to Prism's native AST format, so I wouldn't fret too much over the naming here. Even if we're stuck with it for a while, it's very unlikely we'll support more than a couple of parser engines.